src/NHibernate.sln

```diff
@@ -12,6 +12,7 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		..\build-common\common.xml = ..\build-common\common.xml
 		..\appveyor.yml = ..\appveyor.yml
 		..\ReleaseProcedure.txt = ..\ReleaseProcedure.txt
+		..\src\Directory.Build.props = ..\src\Directory.Build.props
```

Comment:
Do we need it?

Reply:
Why not?

Reply:
nit: Looks like unnecessary noise for me.

Reply:
It helps me not to switch to other editor from VS.

Reply:
In current state it's just used as convenient way to import NHibernate.props to all projects. And that's all. So really there is need to edit it in solution - NHibernate.props should be edited instead.